lib/python/Components/PluginComponent.py

```diff
@@ -56,7 +56,7 @@ def readPluginList(self, directory):
 						except Exception, exc:
 							print "Plugin ", c + "/" + pluginname, "failed to load:", exc
 							# supress errors due to missing plugin.py* files (badly removed plugin)
-							for fn in ('plugin.py', 'plugin.pyc', 'plugin.pyo'):
+							for fn in ('plugin.py', 'plugin.pyo'):
```

Comment:
This should not be changed.

Reply:
As far as I know pyc is ancient and we removed it from PLi and Vision but I can keep it for ATV if you want.